app/views/developments/show/_history.html.haml

```diff
@@ -1,7 +1,7 @@
 .ui.horizontal.divider
 .ui.small.feed
   %h4.ui.header Recent Changes
-  - if @development.recent_history.empty?
+  - if @development.recent_history.empty? && signed_in?
```

Comment:
Users shouldn't need to be signed in to see recent history -- only for the edit link.